articles/concepts/pauli-measurements.md

```diff
@@ -1,120 +1,230 @@
 ---
-title: Pauli Measurements | Microsoft Docs 
+title: Pauli Measurements
 description: Pauli Measurements
 author: QuantumWriter
 uid: microsoft.quantum.concepts.pauli
-ms.author: nawiebe@microsoft.com 
+ms.author: nawiebe@microsoft.com
 ms.date: 12/11/2017
 ms.topic: article
 ---
 
 # Pauli Measurements
 
-In the previous discussions, we have focused on computational basis measurements.  In fact there are other common measurements that occur in quantum computing that, from a notational perspective, are convenient to express in terms of computational basis measurements.  The most common set of these measurements are *Pauli measurements*.  In such cases, it is common to discuss measuring a Pauli operator, in general an operator such as $X,Y,Z$ or $Z\otimes Z, X\otimes X, X\otimes Y$ and so forth.  Discussing measurement in terms of Pauli operators is especially common in the subfield of quantum error correction. In Q# we follow a similar convention; we now explain this alternative view of measurements.
+In the previous discussions, we have focused on computational basis measurements.
+In fact, there are other common measurements that occur in quantum computing that, from a notational perspective, are convenient to express in terms of computational basis measurements.
+As you work with Q#, the most common kind of measurements that you'll run into will likely be *Pauli measurements*, which generalize computational basis measurements to include measurements in other bases, of parity between different qubits, and so forth.
+In such cases, it is common to discuss measuring a Pauli operator, in general an operator such as $X,Y,Z$ or $Z\otimes Z, X\otimes X, X\otimes Y$, and so forth.
```

Comment:
Really like the updated clarification. Just a heads up that now it's a little awkward as both sentences end with ", and so forth."

Reply:
Good point, thanks for catching that. I'll go on and revise accordingly.